### PR TITLE
Remove /env from the README 

### PR DESCRIPTION
**Pull request by taymoork2**
## Description

The node sample was using `ciscospark` instead of `ciscospark/env`,
`ciscospark/env` is the recommended way to use the SDK in a non-browser environment.
However, I suggest that we start to move away (and possible deprecation) from `ciscospark/env` and have people move to just `ciscospark` and have users explicitly `init()` the library.
`/env`'s grabbing of the access token and initializing the SDK "automagically" isn't the best and having two different ways to initiate the SDK is just confusing at the end of the day.
I've included a section that still tells users how to use `ciscospark/env`

I'm also suggesting moving away from the Browserify sample and move to a Parcel sample.

Fixes [38320](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-38320)

## Type of Change

- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by taymoork2** [comment]
`docs/_posts/guides/9999-01-01-getting-started.md` & `documentation/ciscospark.md` both need to be updated because they both include `ciscospark/env`

File: README.md [hunk 1]
```diff
@@ -37,59 +37,101 @@ To use the SDK, you will need Cisco Webex credentials. If you do not already hav
...
-      ciscospark.memberships.create({
-        roomId: room.id,
-        personEmail: `bob@example.com`
-      }),
-    ])
-      .then(() => ciscospark.messages.create({
-        markdown: `**Hi Everyone**`,
-        roomId: room.id
-      }));
+const spark = ciscospark.init({
```
Comment (mattnorris) [inline README.md]: Maybe we should take this opportunity to call this `const teams` instead. Or will that be too confusing with `ciscospark`. What do you think?

File: README.md [hunk 2]
```diff
@@ -37,59 +37,101 @@ To use the SDK, you will need Cisco Webex credentials. If you do not already hav
...
+  credentials: {
+    access_token: process.env.CISCOSPARK_ACCESS_TOKEN
+  }
+});
+
+spark.rooms
```
Comment (mattnorris) [inline README.md]: Why the change from `Promise.all`?

File: README.md [hunk 3]
```diff
@@ -37,59 +37,101 @@ To use the SDK, you will need Cisco Webex credentials. If you do not already hav
...
+      });
+  })
+  .catch(function(error) {
+    console.error(error);
+  });
+
 ```
 
-### Browsers
+### _A note on Browser usage_
```
Comment (mattnorris) [inline README.md]: Lowercase "browser"

File: README.md [hunk 4]
```diff
@@ -37,59 +37,101 @@ To use the SDK, you will need Cisco Webex credentials. If you do not already hav
...
+
+We do not provide a pre-built version of the SDK that includes a `window.ciscospark`.
```
Comment (mattnorris) [inline README.md]: We do not provide a built version of the SDK that includes `window.ciscospark`.

File: README.md [hunk 5]
```diff
@@ -37,59 +37,101 @@ To use the SDK, you will need Cisco Webex credentials. If you do not already hav
...
+If you're already using a bundler (like [Webpack](https://webpack.js.org/) or [Rollup](https://rollupjs.org/)) you can simply import/require the package and use the above snippet and assign the initialized `spark` variable to `window.ciscospark`.
```
Comment (mattnorris) [inline README.md]: If you're already using a bundler like [Webpack](https://webpack.js.org/) or [Rollup](https://rollupjs.org/), you can simply import/require the package, initialize it, and assign it to `window.ciscospark`.

INCLUDE A SMALL SNIPPET HERE SHOWING WHAT YOU JUST DESCRIBED

File: README.md [hunk 6]
```diff
@@ -37,59 +37,101 @@ To use the SDK, you will need Cisco Webex credentials. If you do not already hav
...
+In-browser usage is pretty much the same as Node.js usage, with the addition of handling the user authentication flow for you. See the guide on the [docs site](https://webex.github.io/spark-js-sdk/guides/browsers/) for more information.
```
Comment (mattnorris) [inline README.md]: In-browser usage is almost the same as Node.js, but it handles the user authentication flow for you. See the [browser guide](https://webex.github.io/spark-js-sdk/guides/browsers/) for more information.

File: README.md [hunk 7]
```diff
@@ -37,59 +37,101 @@ To use the SDK, you will need Cisco Webex credentials. If you do not already hav
...
+
+For a quick example on how to use the SDK on a website, we'll be using the [Parcel Bundler](https://parceljs.org/). For any more information and questions on how to use Parcel, please head to their [website](https://parceljs.org/).
```
Comment (mattnorris) [inline README.md]: We will use [Parcel](https://parceljs.org/) to bundle the SDK for a website.

File: README.md [hunk 8]
```diff
@@ -37,59 +37,101 @@ To use the SDK, you will need Cisco Webex credentials. If you do not already hav
...
+
+1. Put the above javascript snippet into a `index.js`
+  a. If you'd like to have `window.ciscospark` available, change the init line to `const spark = window.ciscospark = ciscospark.init`
```
Comment (mattnorris) [inline README.md]: I would rewrite this to just show what the code would look like. Instead of having them all on the same line, break it out into two and use a comment on the bottom line that just says something like, `// Make available to the window.`

File: README.md [hunk 9]
```diff
@@ -37,59 +37,101 @@ To use the SDK, you will need Cisco Webex credentials. If you do not already hav
...
+2. Create a `index.html` with the snippet below
```
Comment (mattnorris) [inline README.md]: Remove "a".

File: README.md [hunk 10]
```diff
@@ -37,59 +37,101 @@ To use the SDK, you will need Cisco Webex credentials. If you do not already hav
...
+3. Run `parcel index.html` in your terminal
+4. Go to [http://localhost:1234](http://localhost:1234) and open the developer console to see the output
```
Comment (mattnorris) [inline README.md]: Put periods at the end of these sentences.

File: README.md [hunk 11]
```diff
@@ -37,59 +37,101 @@ To use the SDK, you will need Cisco Webex credentials. If you do not already hav
...
+ </body>
+</html>
+```
 
-We do not provide a pre-built version of `ciscospark`.
+#### `ciscospark/env`
 
-If you've already got a commonjs or es6 build process in place, you can simply
-use `const ciscospark = require('ciscospark')`.
+`ciscospark/env` is primarily used for the Node.js environment and will continue to be supported, however we suggest that you import/require `ciscospark`.
```
Comment (mattnorris) [inline README.md]: Let's not point people to `ciscospark/env`. I'd rather do things explicitly. 

It may be more appropriate to include a note elsewhere (maybe another doc or .md file) about it and that it will no longer be supported.

File: README.md [hunk 12]
```diff
@@ -37,59 +37,101 @@ To use the SDK, you will need Cisco Webex credentials. If you do not already hav
...
-        personEmail: `bob@example.com`
-      }),
-    ])
-      .then(() => ciscospark.messages.create({
-        markdown: `**Hi Everyone**`,
-        roomId: room.id
-      }));
+const spark = ciscospark.init({
+  credentials: {
+    access_token: process.env.CISCOSPARK_ACCESS_TOKEN
```
Comment (adamweeks) [inline README.md]: Maybe we can put a comment here that the access token doesn't have to come from an env var with this setup.

File: README.md [hunk 13]
```diff
@@ -32,64 +32,124 @@ npm install --save ciscospark
 
 ## Usage
 
+#### _[Still using `ciscospark/env`?](documentation/ciscospark.md#nodejs)_
```
Comment (adamweeks) [inline README.md]: Can you move this to the bottom of the usage section? Let's hide it a bit more.

File: README.md [hunk 14]
```diff
@@ -32,64 +32,124 @@ npm install --save ciscospark
...
-      }),
-    ])
-      .then(() => ciscospark.messages.create({
-        markdown: `**Hi Everyone**`,
-        roomId: room.id
-      }));
-  });
+const teams = ciscospark.init({
+  credentials: {
+    access_token: process.env.ACCESS_TOKEN
```
Comment (adamweeks) [inline README.md]: Let's go with the usage below from your browser usage example. 

```suggestion
    access_token: <your webex teams access token>
```
